src/components/ReportActionAvatars/index.tsx

```diff
@@ -60,6 +60,9 @@ type ReportActionAvatarsProps = {
 
     /** Invited emails to account IDs */
     invitedEmailsToAccountIDs?: InvitedEmailsToAccountIDs;
+
+    /** Whether to use fallback icon */
+    shouldUseFallbackDefaultAvatar?: boolean;
```

Comment:
```suggestion
    shouldUseCustomFallbackAvatar?: boolean;
```

Let's update the prop name as `shouldUseFallbackDefaultAvatar` creates confusion with default fallback avatar(`Expensicons.FallbackAvatar`).